### PR TITLE
Adding a readme and license!

### PR DESCRIPTION
**Pull request by ghost**
Hey! Cool project. I decided to add a readme and a license! :P

### CODE REVIEW

**Comment by Nobelium-Uranium** [comment]
Copyleft? Nice try.

**Comment by Nobelium-Uranium** [comment]
I don't really trust this, sorry.

**Comment by Nobelium-Uranium** [comment]
Generally I don't let people submit PRs unless they notify me on Discord prior to making the PR.

**Comment by ghost** [comment]
> I don't really trust this, sorry.

Please read what I provided in the pull request. I only made a readme file and used a license template provided from Github (GPL3 in particular.) Me adding copyleft is not a trick or a negative thing (and if you think that making proprietary software into free software is a bad thing, I feel bad for you.) I see no good reason to deny users freedom.

A project does need a license in order for it to be considered 'Free Software.' It may not seem important but I can link you a YouTube video that would explain much better than I can.

TLDR: 'Free Software' grants the users of the software 4 essential freedoms. Freedom 1 (Run the software how you want to run the software for whatever purpose.) Freedom 2 (Study and modify the source code.) Freedom 3 (Give or sell copies of the software.) Freedom 4 (Give or sell modified copies of the software.) When software doesn't have a license (even if the source code is available) these freedoms are not present therefore, the software is source-available proprietary. I suggested GPL3 because GPL3 is a strong Free Software license.

I still suggest re-thinking opening up this issue! However, regardless of the outcome, I appreciate you taking a look at this issue.

Here is the link (if you are interested:) https://youtube.com/watch?v=Ag1AKIl_2GM

Free software, free society: Richard Stallman at TEDxGeneva 2014
